lib/ffmpeg/qsv/encoder.py

```diff
@@ -121,7 +121,7 @@ def check_output(self):
       assert m is not None, "Possible incorrect RC mode used"
 
     # lowpower
-    if self.codec not in ["jpeg", "mpeg2"]:
+    if self.codec not in ["jpeg", "mpeg2", "vp9", "vp9-10", "av1", "av1-10"]:
```

Comment:
Why?  Now we will be skipping < MTL check.

Reply:
thanks for finding this, it does not need